src/main/java/igrad/ui/MainWindow.java

```diff
@@ -127,13 +135,15 @@ void displayAvatarSelectionPanel(Model model) {
         resultDisplayPlaceholder.getChildren().add(resultDisplay.getRoot());
         resultDisplay.setFeedbackToUser("Choose an animal guide by entering the NAME of the animal");
 
-        displayStatusPanels(model);
+        displayStatusBar(model);
```

Comment:
I'm removing the status bar, not necessary when selecting the Uwu in the Uwu selection screen